src/Directory.Packages.props

```diff
@@ -41,4 +42,4 @@
     <PackageVersion Include="System.IO.Abstractions.TestingHelpers" Version="17.2.3" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
probably EOL would be required.

Reply:
Could you elaborate? I did not fully understand.

Reply:
oh sorry, I meant an extra line at the end of file is needed.

Reply:
Makes sense. added the EOL